Please help

Hi, this should be asked on StackOverflow, I know, but I am not sure if I get answers there because Google did not find too many wizPurchase hits.

My problem is that nothing seems to happen. I tried to make purchase using debug version of the app and logged the results, hoping I get an error, but nothing. I created signed release version, uploaded to alpha channel, tested using test account, again nothing. Not dialog from Google Play asking for money, no error. I have a div in the page where I log json stringified logs, I use it when I don't have console, in release mode. I tried to write there the success and error responses from wizPurchase.getProductDetails, makePurchase, but nothing appears.

I will paste here some of my code, maybe you find the problem:

```javascript
import store from '../store'
var dispatch = store.dispatch

var StoreHolder = {}
;(function() {
  window.addEventListener('load', ev => {
    document.addEventListener(
      'deviceready',
      deviceready => {
        console.log('window in deviceready', window)
        init()
      },
      false
    )
  })
})()

function init() {
  dispatch('insertLog', { timeStamp: new Date(), info: 'init window.wizPurchase', details: window.wizPurchase })
  StoreHolder.wizPurchase = window.wizPurchase
}

export default StoreHolder
```

Then in my Vue.js app I check vor `wizPurchase` being present and only then I call its methods:

```javascript
    import StoreHolder from './modules/store'
    //...
    test(id) {
      this.insertLog({ timeStamp: new Date(), info: 'test call', details: StoreHolder })

      if (StoreHolder.wizPurchase) {
        this.insertLog({ timeStamp: new Date(), info: 'test wizPurchase', details: StoreHolder.wizPurchase })
        // I get to this place, StoreHolder.wizPurchase exists and it is logged

        //below code seems to be doing nothing at all, also no error
        StoreHolder.wizPurchase.getProductDetails(
          id,
          success => {
            this.insertLog({ timeStamp: new Date(), info: 'test call', details: success })
            console.log('success', success)
          },
          error => {
            this.insertLog({ timeStamp: new Date(), info: 'test call', details: error })
            console.log('error', error)
          }
        )
      }
    }
```

Thank you for your help!

After removing and adding the android platform, running `cordova run android` I get error that InAppBillingService is missing and can't be imported.
```
cordova\platforms\android\app\src\main\java\com\smartmobilesoftware\util\IabHelper.java:32: error: package com.android.vending.billing does not exist
import com.android.vending.billing.IInAppBillingService;
```
I searched for `IInAppBillingService` inside android folder and I found it at path `\cordova\platforms\android\src\com\android\vending\billing\IInAppBillingService.aidl`.

A Google search pointed me to this fix for the above error, which says to take the file and place it in another folder `cordova\platforms\android\app\src\main\aidl\com\android\vending\billing\`
The place where I found the solution for the error: https://github.com/AlexDisler/cordova-plugin-inapppurchase/issues/196

After fixing that, I am back at square one, meaning it can't load the billing key from the config.xml file.
Logcat: `W/System.err:     at jp.wizcorp.wizpurchase.WizPurchase.initialize(WizPurchase.java:119)`

--- a/plugin.xml
+++ b/plugin.xml
@@ -46,13 +46,13 @@
 	<platform name="android">
 		<preference name="BILLING_KEY" />
 
-		<config-file target="res/xml/config.xml" parent="/*">
+		<config-file target="app/src/main/res/xml/config.xml" parent="/*">
 			<feature name="WizPurchase">
 				<param name="android-package" value="jp.wizcorp.wizpurchase.WizPurchase" />
 			</feature>
 		</config-file>
 
-		<config-file target="AndroidManifest.xml" parent="/manifest">
+		<config-file target="app/src/main/AndroidManifest.xml" parent="/manifest">
 			<uses-permission android:name="com.android.vending.BILLING" />
 		</config-file>
 
@@ -71,11 +71,11 @@
 		<source-file src="src/android/com/smartmobilesoftware/util/SkuDetails.java" target-dir="src/com/smartmobilesoftware/util" />
 
 		<!-- Billing Classes -->
-		<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="src/com/android/vending/billing" />
+		<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="app/src/main/aidl/com/android/vending/billing" />
 
 		<!-- Billing Key String Holder -->
 		<resource-file src="src/android/res/values/billing_key.xml" target="res/values/billing_key.xml" />
-		<config-file target="res/values/billing_key.xml" parent="/*">
+		<config-file target="app/src/main/res/values/billing_key.xml" parent="/*">
 			<string name="billing_key">$BILLING_KEY</string>
 		</config-file>
 	</platform>

